samples/v1.5/Scenarios/InputsWithValidation.json

```diff
@@ -1,7 +1,7 @@
 {
 	"$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
 	"type": "AdaptiveCard",
-	"version": "1.0",
+	"version": "1.5",
```

Comment:
`1.3`?

Reply:
We use `tooltip` within this one on `Action.OpenUrl`